.eslintrc.json

```
@@ -0,0 +1,15 @@
+{
+  "env": {
+    "node": true,
+    "es2021": true
+  },
+  "extends": ["eslint:recommended", "plugin:@typescript-eslint/recommended", "pretter"],
```

タイポ！！！！！！！！！！！！！！！！！！
```suggestion
  "extends": ["eslint:recommended", "plugin:@typescript-eslint/recommended", "prettier"],
```